domain/timeline/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/timeline/service/TimelineService.kt

```diff
@@ -34,6 +36,13 @@ class TimelineService(
    * Returns the [Timeline] for the prisoner identified by their prison number. Otherwise, throws
    * [TimelineNotFoundException] if it cannot be found.
    */
-  fun getTimelineForPrisoner(prisonNumber: String): Timeline =
-    persistenceAdapter.getTimelineForPrisoner(prisonNumber) ?: throw TimelineNotFoundException(prisonNumber)
+  fun getTimelineForPrisoner(prisonNumber: String): Timeline {
+    val prisonerTimeline =
+      persistenceAdapter.getTimelineForPrisoner(prisonNumber) ?: throw TimelineNotFoundException(prisonNumber)
+
+    if (callPrisonApiEnabled) {
+      prisonerTimeline.addEvents(prisonTimelineService.getPrisonTimelineEvents(prisonNumber))
```

Comment:
We simply add additional TimelineEvents here, which will ended up being sorted by date, along with the other existing events.